Cargo.toml

```diff
@@ -105,6 +105,9 @@ fuel-types = "0.59"
 fuel-tx = "0.59"
 fuel-vm = "0.59"
 
+# Dependencies for Fuel Telemetry
+fuel-telemetry = { path = "../fuel-telemetry", version = "0.1" }
```

Comment:
This should not be a local path

Reply:
Cool, updated to point to Github. Thanks again!